packages/identity/src/identity/ed25519.test.ts

```diff
@@ -64,14 +64,29 @@ test('DER decoding of invalid keys', async () => {
   }).toThrow();
 });
 
-test('fails with improper seed', () => {
-  expect(() => Ed25519KeyIdentity.generate(new Uint8Array(new Array(31).fill(0)))).toThrow();
-  expect(() => Ed25519KeyIdentity.generate(new Uint8Array(new Array(33).fill(0)))).toThrow();
+test('fromSecretKey should generate an identity', () => {
```

Comment:
Thanks for adding these tests!

Should we keep the old test `fails with improper seed` since the seed is still supported for Ed25519?